pvlib/spectrum/mismatch.py

```diff
@@ -0,0 +1,164 @@
+"""
+The ``mismatch`` module provides functions for spectral mismatch calculations.
+"""
+
+import pvlib
+import numpy as np
+import pandas as pd
+from scipy.interpolate import interp1d
+import os
+
+
+def get_sample_sr(wavelength=None):
```

Comment:
Agreed `get_example_spectral_response` would be good.

Reply:
I have no problems making a name and parameter change to pass the review but I think it would be inappropriate to put additional technology options into this function in the future.  I believe SR data should be in data files in general.